frontend/projects/ui/src/app/components/form-object/form-object.component.ts

```diff
@@ -63,6 +64,37 @@ export class FormObjectComponent {
   ) {}
 
   ngOnInit() {
+    this.setDisplays()
+
+    // setTimeout hack to avoid ExpressionChangedAfterItHasBeenCheckedError
+    setTimeout(() => {
+      if (this.original) {
+        Object.keys(this.current || {}).forEach(key => {
+          if ((this.original as Config)[key] === undefined) {
+            this.hasNewOptions.emit()
+          }
+        })
+      }
+    }, 10)
```

Comment:
You don't need any time, just default 0 is enough to move it to the next frame.

Reply:
fixed